yjit/Cargo.toml

```diff
@@ -20,6 +20,9 @@ crate-type = ["staticlib"]
 # depending on this crate.
 capstone = { version = "0.10.0", optional = true }
 
+[build-dependencies]
+cc = "1.0"
+
```

Comment:
We're pretty strict about wanting not to add dependencies on external crates. The solution is to put the definitions you need to export in `yjit.c`.